docs/user/building-images.md

```diff
@@ -68,7 +68,7 @@ The following procedure describes the early binding method of building the agent
 Use the `flightctl` CLI to authenticate with the Flight Control service, then run the following command to obtain enrollment credentials with a validity of one year, in the format of an agent configuration file:
 
 ```console
-flightctl certificate request --signer=enrollment --validity=1y -o agent-config > config.yaml
+flightctl certificate request --cert-type=enrollment --name=client-enrollment --expiration=365d --output-format=embedded > config.yaml
```

Comment:
This names the files as well as the CSR resource itself, so it would make it easier to find later if something in the docs references the `client-enrollment` CSR. But I don't think it's critical, so I could remove it.